src/peers/PeersTable/PeersTable.css

```diff
@@ -0,0 +1,4 @@
+
+.PeersTableContainer {
+  height: 220px;
```

Comment:
Minor, but I'd just use an inline style attributes for this as the value is duplicated in the definition of the table. I'd inline `.MapContainer` as well, just to keep all the definitions together.

Reply:
Agree, and that way we can delete those two CSS files 👍